plugins/paytovote/paytovote.go

```diff
@@ -0,0 +1,183 @@
+package paytovote
+
+import (
+	"fmt"
+
+	abci "github.com/tendermint/abci/types"
+	"github.com/tendermint/basecoin/types"
+	"github.com/tendermint/go-wire"
+)
+
+const (
+	TypeByteCreateIssue byte = 0x00
+	TypeByteVoteFor     byte = 0x01
+	TypeByteVoteAgainst byte = 0x02
+	TypeByteVoteSpoiled byte = 0x03
+)
+
+type P2VPluginState struct {
```

Comment:
aha good eye. P2VIssueState? either/or I agree that this makes more sense than pluginState